README.md

```diff
@@ -1049,6 +1088,25 @@ On OS X and Linux, your git configuration file is stored in ```~/.gitconfig```.
     zap = fetch -p
 ```
 
+<a name="adding-empty-repository"></a>
+### I want to add an empty directory to my repository
+
+You can’t! Git doesn’t support this, but there’s a hack. You can create a .gitignore file in the directory with the following contents:
+
+```
+ # Ignore everything in this directory
+ *
+ # Except this file
+ !.gitignore
+```
+
+Another common convention is to make an empty file in the folder, titled .gitkeep.
```

Comment:
Hi @Rijul1999, I propose to rename this file `.keep`, because:
- `.gitkeep` suggests this filename is a feature of Git (e.g. like .gitignore), which is not the case
- Git is not the only SCM that does not track empty folders, so the `.keep` filename sounds more SCM-agnostic
- and finally the convention to name this dummy file `.keep` is also widespread.

See for example:
- https://www.git-tower.com/learn/git/faq/add-empty-folder-to-version-control
- https://medium.com/@kinduff/hey-git-please-keep-those-folders-eb0ed37621c8
- https://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep?rq=1#comment30099040_7229996

Reply:
There are almost five million hits for `.gitkeep` on GitHub: https://github.com/search?utf8=%E2%9C%93&q=filename%3A%22.gitkeep%22&type=. Double that for `. keep`. I think they're both valid - maybe add a note saying you can name the file as either?

Reply:
Indeed there are around five million `.gitkeep` files and around eleven million `.keep` files on GitHub :) so even if `.keep` wins over `.gitkeep`, one can also mention both conventions...

Reply:
Alright, I'll add a note for .keep